python/lsst/verify/ap/verify_ap.py

```diff
@@ -0,0 +1,136 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+from __future__ import absolute_import, division, print_function
+
+__all__ = []
+
+import argparse
+import os
+import re
+
+import lsst.log
+from lsst.verify.ap.dataset import Dataset
+from lsst.verify.ap.metrics import MetricsParser, check_squash_ready, AutoJob
+from lsst.verify.ap.pipedriver import ApPipeParser, run_ap_pipe
+
+
+class _VerifyApParser(argparse.ArgumentParser):
```

Comment:
I'm not yet up to speed on the verify framework or the verify_ap design discussions, but is there any reason that this stuff is not set up as a `lsst.pipe.base.Task` subclass? That way you'd already have argument and config file parsing built-in.